classes/plugins/event.py

```diff
@@ -0,0 +1,56 @@
+import os
+import builtins
+import logging
+import pickle
+import typing
+_pickleLevel_ = 5 #pickle level to use when comprssing and storing events
+def _save_(events):
+    folder = os.path.dirname(os.path.realpath(__file__))
+    with open(f'{folder}/events.pickle',"wb") as file:
+        pickle.dump(events,file,protocol=_pickleLevel_)
+
+def _load_():
```

Comment:
Rename this method or make it just assign `_events_` and not return anything. Currently the name and the use conflict with each other